packages/client/src/utils/FclConfig.js

```diff
@@ -0,0 +1,14 @@
+import * as fcl from '@onflow/fcl';
+
+fcl.config({
+  'flow.network': 'testnet',
+  'app.detail.title': 'Wuphf moments',
+  'accessNode.api': 'https://rest-testnet.onflow.org',
+  'app.detail.icon': 'https://placekitten.com/g/200/200',
+  'discovery.wallet':
+    'https://fcl-discovery.onflow.org/testnet/authn',
+});
+
+export const getFclConfiguration = () => {
+  return fcl;
+};
```

Comment:
Not exactly needed now but we could use [var envs](https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env) to set this config, so it can be easily changed between various environment. 
I will use one in my next commit, to show you how it works.